dynamic_instrumentation/src/point/mod.rs

```diff
@@ -86,23 +86,23 @@ pub struct CollectInstrumentationPoints<'a, 'tcx: 'a> {
     pub body: &'a Body<'tcx>,
     pub instrumentation_points: Vec<InstrumentationPoint<'tcx>>,
     assignment: Option<(Place<'tcx>, Rvalue<'tcx>)>,
-    pub addr_taken_local_substitutions: IndexMap<Local, Local>,
+    pub local_to_address: IndexMap<Local, Local>,
```

Comment:
I think the `addr_taken` part of the name was useful here since this is in `CollectInstrumentationPoints`, which doesn't already mention address taken like the other ones.